### DISCUSSION TITLE
Building materials and roof orientation

### DISCUSSION THREAD

**Discussion by beecho01** [streetcomplete/StreetComplete#4295] [Ideas]
I've been working on importing my town into Osm. I use F4Maps to visually identify if I have missed any "roof:material" and "building:material" tags. Would be super helpful to identify and set these in app.

Furthermore, sometimes I have noted that the default "roof:orientation" for a roof is along the length of a building (front to back). In some cases this is not true and the roof peak goes across the shorter building length (side to side). Would be useful to set this as a quest also.

**Comment by HolgerJeromin** [comment]
ref #1358 (rejected material and color) and #1063 (waiting for upstream)

[overpass turbo](https://overpass-turbo.eu) could be useful for you, too:
```
[out:json][timeout:25];
// gather results
(
  // query part for: “building=* and "roof:material"!=*”
  node["building"]["roof:material"!~".*"]({{bbox}});
  way["building"]["roof:material"!~".*"]({{bbox}});
  relation["building"]["roof:material"!~".*"]({{bbox}});
);
// print results
out body;
>;
out skel qt;
